### PR TITLE
Tie not confirmed sightings with frontend

### PR DESCRIPTION
**Pull request by LenaESoftwire**
https://trello.com/c/KrygV8Rb/62-tie-together-get-not-confirmed-sightings-with-frontend

### CODE REVIEW

File: WhaleSpotting/Controllers/SightingsController.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 namespace WhaleSpotting.Controllers
 {
     [ApiController]
-    [Route("[controller]")]
+    [Route("api/[controller]")]
```
Comment (oskarwilliams) [inline WhaleSpotting/Controllers/SightingsController.cs]: is this part of this ticket

File: WhaleSpotting/react-app/tsconfig.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
     "lib": [
       "dom",
       "dom.iterable",
-      "esnext"
+      "esnext",
+      "ES2015"
```
Comment (oskarwilliams) [inline WhaleSpotting/react-app/tsconfig.json]: Remove this please

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/Controllers/SightingsController.cs [hunk 3]
```diff
@@ -59,6 +59,7 @@ public async Task<ActionResult<SightingResponseModel>> ConfirmSighting([FromRout
             return sighting == null ? NotFound() : sighting;
         }
 
+        //TODO add admin role
```
Comment (oskarwilliams) [inline WhaleSpotting/Controllers/SightingsController.cs:64]: This todo is now merged, so we can add the actual code now

**Review by oskarwilliams — CHANGES_REQUESTED** [review]

File: WhaleSpotting/Controllers/SightingsController.cs (hunk 3 above)
Comment (francescastocco) [reply WhaleSpotting/Controllers/SightingsController.cs:64]: we've done this on our pagination ticket - is that ok?

**Review by oskarwilliams — APPROVED** [review]

**Pull request merged**